nbconvert/exporters/export.py

```diff
@@ -31,9 +32,14 @@
 #-----------------------------------------------------------------------------
 
 def DocDecorator(f):
-    
+    """
+    Deprecated since version 5.0.
```

Comment:
should we explicitly make it a private API then?  E.g., `_DocDecorator`? 
Or is there a reason to not add underscores to decorators?

Also, is it following best practices to indicate that something is not a public API by not including documentation? Shouldn't it be that we make things that are not public APIs explicitly private, but also include documentation so that when new core developers look at the function/class/&c. , they don't need to reverse engineer it? 

Also, I don't entirely understand the convention around naming in this case…this isn't a class but a decorator function, so then why do we capitalise it? I believe there's a reason I just don't know what it is.